src/components/Alert/Alert.scss

```diff
@@ -38,6 +38,10 @@ $c-dark-alert-bg-color: $color-gray-700 !default;
 	> .c-spinner {
 		margin: 5px 12px 4px 0;
 	}
+
+	> .o-svg-icon {
```

Comment:
dit fixed mogelijks ook deze PR:
https://github.com/viaacode/avo2-components/pull/177

Reply:
aahja 😄was iets da mij was opgevallen in de toast messages